### PR TITLE
Add README-docker-compose.md and reference it from README.md and dock…

### PR DESCRIPTION
**Pull request by cwisecarver**
…er-compose.yml

### CODE REVIEW

**Comment by cwisecarver** [comment]
@icereval CR for me? I think it's all right. Links work, seems like English.

File: README-docker-compose.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  - OSX: https://www.docker.com/products/docker#/mac
+  - Windows: https://www.docker.com/products/docker#/windows
+2. Grant the docker client additional memory and cpu (minimum of 4GB and 2 CPU)
+   - OSX: https://docs.docker.com/docker-for-mac/#/preferences
+   - Windows: https://docs.docker.com/docker-for-windows/#advanced
+3. Setup the Operating System
+  - OSX
+    - Alias the loopback interface
+
+      `$ sudo ifconfig lo0 alias 192.168.168.167`
```
Comment (icereval) [inline README-docker-compose.md]: can you add a big note about running this on reboot

**Comment by icereval** [comment]
Does this replace README.md? if not you'll want to update the `Running the OSF` section.

**Comment by cwisecarver** [comment]
I added it [here](https://github.com/cwisecarver/osf.io/blob/feature/update-docker-compose-readme/README.md#table-of-contents), above `Running the OSF` and changed `Running the OSF` to `Running the OSF locally`.

**Comment by cwisecarver** [comment]
I made it more explicit. https://github.com/cwisecarver/osf.io/blob/feature/update-docker-compose-readme/README.md#running-the-osf

**Comment by cwisecarver** [comment]
Still need to add something about migrations, then I'm gtg.

**Pull request closed without merging**